examples/websocket_client.py

```
@@ -0,0 +1,282 @@
+            # Send leave message
+            leave_message = {
+                "topic": "killmails:lobby",
+                "event": "phx_leave",
+                "payload": {},
+                "ref": 5
+            }
+            
+            try:
+                await self.websocket.send(json.dumps(leave_message))
+                await self.websocket.close()
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)